Extract vocabularies from the specs (#1510)
json-schema-org/json-schema-spec: jsonschema-core.md

### Loading Instance Data

While none of the keywords defined as part of this or the associated
documents define a keyword which target and/or load instance data, it is
possible that extensions may wish to do so.

Keywords MAY be defined to use JSON Pointers or Relative JSON Pointers to
examine parts of an instance outside the current evaluation location.

Keywords that allow adjusting the location using a Relative JSON Pointer SHOULD
default to using the current location if a default is desireable.

## The JSON Schema Core Keywords {#core}

Keywords declared in this section, which all begin with "$", are essential to
processing JSON Schema. These keywords inform implementations how to process any
schema or meta-schema, including those split across multiple documents, or exist
to reserve keywords for purposes that require guaranteed interoperability.

Support for these keywords MUST be considered mandatory at all times in order to

> bootstrap the processing of further keywords.

Author: I see the `$` keywords as "directive" keywords, meaning they tell you how to process and navigate the schema.  In that way, they are sort of "bootstrapping", but I'm open to massaging it a bit.

> Support for these keywords MUST be considered mandatory at all times in order to
> bootstrap the processing of further keywords.

Reviewer: I see what you're trying to say. Maybe something like this,

```suggestion
Support for these keywords MUST be considered mandatory at all times as they are
necessary to navigate and process any schema.
```

The "$" prefix is reserved for use by this specification. Extensions MUST NOT
define new keywords that begin with "$".

### Meta-Schemas

Meta-schemas are used to inform an implementation how to interpret a schema.
Every schema has a meta-schema, which can be explicitly declared using the

> `$schema` keyword.

Reviewer: I think this section was effectively the introduction to and definition of the term "dialect". That ends up getting lost in the changes. Do we want to do something to include that defines "dialect"?

Author: I'm not sure.  I wonder if that's something that needs to be re-introduced with the proposal.

Reviewer: I think the concept of a dialect should stay. The concept isn't dependent on the vocabulary system. I explain further in my latest review.

Author: How do you want to define "dialect"?  Is it merely the collection of keywords?  Open to suggestions.
